smurfs/src/actions/index.js

```diff
@@ -0,0 +1,34 @@
+import axios from 'axios';
+
+export const FETCHING = "FETCHING"
+export const FETCHING_SUCCESS = "FETCHING_SUCCESS"
+export const FETCHING_FAILED = "FETCHING_FAILED"
+
+export const ADD_SMURF = "ADD_SMURF"
```

Comment:
I usually try to keep your naming convensions the same across type groupings. 'FETCHING_START' / 'ADD_SMURF_START'